packages/@misk/core/src/features/Navbar/MiskContainer.tsx

```diff
@@ -58,7 +63,8 @@ export const MiskNavbarContainer = (
     menuShowButton,
     navbar_items,
     status,
-    theme = defaultTheme
+    theme = defaultTheme,
+    items_alignment = alignment
```

Comment:
Try `navbar_items_alignment = Alignment.Left` as an inline setting default parameter. I think it will work though please double check that if you use it with `Alignment.Right` it correctly passes through.

Reply:
Probably can leave this just as `navbar_items_alignment` with no default assignment because BlueprintJS provides sane defaults for us.